### ISSUE TITLE
Can't use the side page button

### ISSUE THREAD

**Issue by xiaoxu777** [koreader/koreader#11555]
* KOReader version:
* Device:KindleVoyage

#### Issue
I can't use the side page button. 
![tempImageb0JkVr](https://github.com/koreader/koreader/assets/54737217/c3ee09f4-ca2c-4ce3-a3a0-e329ed713fba)

**Comment by xiaoxu777** [comment]
The functionality was restored after I set up a detailed log. But for some reason the force feedback is weak

**Comment by hius07** [comment]
The feedback level is adjusted in the native settings, not in KOReader.

**Comment by Hzj-jie** [comment]
If you have ads screensaver, the buttons won't work after resuming from sleep. It's a known issue since the ads hijacks the button inputs. You can however work around it with a hacky way.

**Comment by Atul-86** [comment]
> If you have ads screensaver, the buttons won't work after resuming from sleep. It's a known issue since the ads hijacks the button inputs. You can however work around it with a hacky way.

I have the same problem. Whats the hacky way?

**Comment by Commodore64user** [comment]
> > If you have ads screensaver, the buttons won't work after resuming from sleep. It's a known issue since the ads hijacks the button inputs. You can however work around it with a hacky way.
> 
> I have the same problem. Whats the hacky way?

I am afraid you will not find it in these shores... I wish you the best in your search, and may you all find the 'hacky way'

**Comment by mergen3107** [comment]
Should this be closed?

For non ad Voyage KOReader supports buttons. 

Everything else (ads, anti-ads and button blocking) is out of scope.

**Comment by Frenzie** [comment]
If I understand correctly the problem is that buttons no longer work after the ads have been shown. That's not by definition out of scope as part of platform support (but I have no interest in it ;-).

**Comment by mergen3107** [comment]
No, KOReader doesn't intervene in the ads engine at all. If the firmware shoots itself in the foot when ads show up and makes the buttons stop working - well, this is defo out of scope :D

**Comment by johan456789** [comment]
> I have the same problem. Whats the hacky way?

@Atul-86 Before you try the hacky way, try contact Amazon support. A few years ago I tried to pay to remove special offers and it errored out. After contacting support, they just removed it for free for me. (I forgot if it was paperwhite 1 or voyage or both)
